core/src/main/java/org/apache/calcite/materialize/LatticeSuggester.java

```diff
@@ -334,7 +334,7 @@ private Lattice findMatch(final Lattice lattice, MutableNode mutableNode) {
       }
 
       if (bestMatch != null) {
-        // Fix up the best batch
+        // Fix up the best match
         for (Path path
```

Comment:
There is already a fix there, remove this change or keep credit for the contributor.

Reply:
Will do.